src/lib/pages/module-details/components/ModuleInfoBody.tsx

```diff
@@ -0,0 +1,119 @@
+import { Flex, Grid } from "@chakra-ui/react";
+
+import { ExplorerLink } from "lib/components/ExplorerLink";
+import type { LabelTextProps } from "lib/components/LabelText";
+import { LabelText } from "lib/components/LabelText";
+import { dateFromNow, formatUTC } from "lib/utils";
+
+import type { ModuleInfoProps } from "./ModuleInfo";
+
+interface ModuleInfoBodyValueProps {
+  labelText: Pick<LabelTextProps, "label" | "helperText1" | "helperText2">;
+  content: string | JSX.Element;
+}
+
+const ModuleInfoBodyValue = ({
+  labelText,
+  content,
+}: ModuleInfoBodyValueProps) => (
+  <Flex flexDirection="column" gap={1}>
```

Comment:
Is Flex necessary here? If not, remove this component and use LabelText directly below